packages/gatsby-plugin-vanilla-extract/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "gatsby-plugin-vanilla-extract",
+  "version": "2.0.0",
+  "description": "Gatsby plugin which wraps the vanilla-extract package for easy use",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/KyleAMathews/gatsby-plugin-vanilla-extract.git"
```

Comment:
URL incorrect

Reply:
See the contributor docs for information on this formatting. Or another plugin.